src/overlays/actors/ovl_Obj_Usiyane/z_obj_usiyane.c

```diff
@@ -28,40 +28,254 @@ const ActorInit Obj_Usiyane_InitVars = {
     (ActorFunc)ObjUsiyane_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80C086B0[] = {
+typedef struct {
+    /* 0x00 */ Vec3f unk_00;
+    /* 0x00 */ Vec3s unk_0C;
```

Comment:
```suggestion
    /* 0x0C */ Vec3s unk_0C;
```
Also this whole struct looks like it could just be PosRot, but maybe not depending on context. Can you just add a todo to figure out if `PosRot` makes sense for this.